app/javascript/controllers/application_controller.js

```diff
@@ -0,0 +1,63 @@
+"use strict"
+
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  static targets = ["newMessageField"]
+
+  connect() {
+    document.addEventListener("keydown", this.keypress.bind(this), false)
+  }
+
+  disconnect() {
+    document.removeEventListener("keydown", this.keypress.bind(this))
+  }
+
+  keypress(event) {
```

Comment:
I looked at Stimulus-Use's Hotkey library for this and honestly it felt totally unnecessary. 

This is fundamental javascript and we want it to be as fast as possible -- importing several hundred lines of meta-dispatching based on string compositions representing key chords just to accomplish some basic switching feels unnecessary.

However, if we end up in a place where we allow users to reconfigure shortcuts, I'd vote we move for that package.